buildbot-conf/cordova.conf

```diff
@@ -229,6 +229,8 @@ plugins_cleanup_steps = [
     NPMInstall(command=['shelljs'], what='shelljs'),
     RMRF('~/.cordova/*', description='removing cache', haltOnFailure=False),
     RMRF(BASE_WORKDIR + '/*', description='cleaning workspace', haltOnFailure=False),
+    RMRF(BASE_WORKDIR + '/*', description='cleaning workspace again', haltOnFailure=False),
+    RMRF(BASE_WORKDIR + '/*', description='cleaning workspace yet again', haltOnFailure=True),
```

Comment:
Both are functionally equivalent. I think the three repeated steps are a bit more clear, since it's very easy to see that it's repeated, and they make it evident on the waterfall page that this is something that still needs to be fixed properly.

Also, we haven't seen this issue for `~/.cordova/*`.